addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -8,6 +8,10 @@
       "playSound": false
     }
   },
+  "scripts": {
+    "start": "ng server",
+    "postinstall": "typings install --ambient"
```

Comment:
you should test what happens when a typing is removed from typings.json - make sure that this will remove it from the typings dir. otherwise contributors end up with stale extra typings files in their working dir